Dockerfile

```diff
@@ -40,9 +40,9 @@ RUN wget https://download.docker.com/linux/debian/gpg \
     && rm -rf /var/lib/apt/lists/*
 
 # Datadog
-RUN curl -L -o /tmp/datadog-php-tracer.deb https://github.com/DataDog/dd-trace-php/releases/download/${DD_PHP_TRACER_VERSION}/datadog-php-tracer_${DD_PHP_TRACER_VERSION}_amd64.deb \
- && dpkg -i /tmp/datadog-php-tracer.deb \
- && rm /tmp/datadog-php-tracer.deb
+RUN curl -LO "https://github.com/DataDog/dd-trace-php/releases/download/${DD_PHP_TRACER_VERSION}/datadog-setup.php" > /tmp/datadog-setup.php \

```

Comment:
Hele super iniciativa, ale nez tohle nasadime, je potreba vypnout profiling na runneru :grin: https://github.com/keboola/job-queue-daemon/blob/0340fa6e16ddd76ca2145312f88da67edacd1df3/src/Cluster/JobPodSpecification.php#L303

Reply:
muzu to pribalit k tou billingu, jestli chces.. stejne budu muset udelat jeste nejake upravy

Reply:
A ten profiling nefunguje s tou novou verzi? Nebo v cem je problem?

Reply:
To by melo fungovat prave si mysli, nebo prave proto, ze to zacne fungovat tak to chces vypnout? :D

Tak to plz vypni v tom demonovi, tohle zatim pocka v pohode.

Reply:
Problem je, ze touhle "novou instalaci" se nainstaluje i profiling extension (`--enable-profiling`, coz chceme), takze kdyz je v ENV nastavene `DD_PROFILING_ENABLED: true`, zacne to hned odesilat profiling do DD, coz nechceme (je to drahy).

Idelani stav je:
* nainstalovana DD extension s povolenym vsim (vc. profilingu)
* profiling by default vypnuty (`DD_PROFILING_ENABLED: false`, nebo to do ENV vubec nedavat)
* kdyz budeme chtit neco profilovat, docesne si to zapnout nastavenim `DD_PROFILING_ENABLED: true`

Reply:
Jasne, diky, tak kdyz se mergne demon, tak muzem pak tenhle PR taky mergnout